documentation/gitbook/gempyor/model-implementation/specifying-initial-conditions.md

```diff
@@ -1,192 +1,341 @@
-
-`initial_conditions:initial_file_type` Only required for `method: “FolderDraw”`. Description TBA
-
-`initial_conditions::allow_missing_subpops` Optional for all methods, determines what will happen if `initial_conditions_file` is missing values for some subpopulations. If FALSE, the default behavior, or unspecified, an error will occur if subpopulations are missing. If TRUE, then for subpopulations missing from the `initial_conditions` file, it will be assumed that all individuals begin in the first compartment (the “first” compartment depends on how the model was specified, and will be the compartment that contains the first named category in each compartment group), unless another compartment is designated to hold the rest of the individuals ;
-
-`initial_conditions::allow_missing_compartments` Optional for all methods. If FALSE, the default behavior, or unspecified, an error will occur if any compartments are missing for any subpopulation. If TRUE, then it will be assumed there are zero individuals in compartments missing from the `initial_conditions file`.
-
-`initial_conditions::proportional` If TRUE, assume that the user has specified all input initial conditions as fractions of the population, instead of numbers of individuals (the default behavior, or if set to FALSE). Code will check that initial values in all compartments sum to 1.0 and throw an error if not, and then will multiply all values by the total population size for that subpopulation ;
-
-Details on implementing each initial conditions method and the options that go along with it are below.
-
-### `initial_conditions::method`
+```yaml
+initial_conditions:
+  method: Default
+```
 
-#### Default
+Further configuration placed under the `initial_conditions` section is passed on to the method used and depends on the method. The methods included with `gempyor` are 'Default', 'SetInitialConditions', 'SetInitialConditionsFolderDraw', 'FromFile', 'InitialConditionsFolderDraw'.
 
-The default initial conditions are that the initial value of all compartments for each subpopulation will be zero, except for the first compartment, whose value will be the population size. The “first” compartment depends on how the model was specified, and will be the compartment that contains the first named category in each compartment group.
+### 'Default' Method
 
-For example, a model with the following compartments
+The 'Default' initial conditions will place the entire subpopulation into the first compartment specified by the model. For example:
 
-```
- compartments:
-   infection_stage: ["S", "I", "R"]
-   age_group: ["child", "adult"]
-   vaccination_status: ["unvaxxed", "vaxxed"]
+```yaml
+compartments:
+  infection_stage: ["S", "I", "R"]
  
- initial_conditions:
-   method: default
+initial_conditions:
+  method: default
```

Comment:
i'm fine w/ caps vs not (prefer not, but for now let's prioritize minimizing changes)

```suggestion
  method: Default
```